env.example

```diff
@@ -89,7 +89,7 @@ TZ=UTC
 #WHITEBOARD_COLLAB_SERVER_URL_BASE=http://whiteboard.meet.jitsi
 
 # Set the excalidraw-backend public URL (uncomment to enable)
-#WHITEBOARD_COLLAB_SERVER_PUBLIC_URL=https://whiteboard.meet.my.domain
+#WHITEBOARD_COLLAB_SERVER_PUBLIC_URL=http://127.0.0.1:3002
```

Comment:
This is for the external version, if you want to use the existing one in whiteboard.yml then uncometing the one above is enough.